src/campaigns.ts

```diff
@@ -421,13 +421,8 @@ export const campaigns: Campaign[] = [
             endTimestampExclusive,
             rewardAmount: '25000',
             excludedReferrers: {},
-            previousResultDirectories: [
-              new ResultDirectory({
-                datadir: 'rewards',
-                name: 'celo-pg-s1',
-                startTimestamp: new Date('2025-08-26T00:00:00Z'),
-                endTimestampExclusive: new Date('2025-09-09T00:00:00Z'),
-              }),
+            previousKpiFiles: [
+              'https://storage.googleapis.com/divvi-campaign-data-production/kpi/celo-pg-s1/2025-08-26T00:00:00.000Z_2025-09-09T00:00:00.000Z/kpi.json',
```

Comment:
FYI, i'll make this a little nicer and fill our the other previousKpiFile properties in a follow-on PR